### PR TITLE
Fix failing GroupByEntityOutputSpec

### PR DESCRIPTION
**Pull request by raronson**
@charleso @markhibberd Sane issue as #710 but with `GroupByEntityOutputSpec`

### CODE REVIEW

**Comment by charleso** [comment]
:+1:

File: ivory-operation/src/test/scala/com/ambiata/ivory/operation/extraction/output/GroupByEntityOutputSpec.scala [hunk 1]
```diff
@@ -43,7 +43,7 @@ class GroupByEntityOutputSpec extends Specification with ThrownExpectations with
     RepositoryBuilder.using(createDense(fwd.facts, fwd.dictionary,
       GroupByEntityFormat.DenseText(Delimiter.Psv, "NA", TextEscaping.Delimited, TextFormat.Json)) {
       (_, file) =>
-        IvoryLocation.readLines(file).map(l => parse(fwd.dictionary, "NA", l.map(EavtParsers.splitLine('|', _))))
+        IvoryLocation.readLines(file).map(l => parse(fwd.dictionary, "NA", l.map(_.split("\\|", -1).toList)))
```
Comment (markhibberd) [inline ivory-operation/src/test/scala/com/ambiata/ivory/operation/extraction/output/GroupByEntityOutputSpec.scala]: We should remove that from mundane, it is just broken I think.

Comment (markhibberd) [inline ivory-operation/src/test/scala/com/ambiata/ivory/operation/extraction/output/GroupByEntityOutputSpec.scala]: Or at least make it clear what is happening somehow.

There used to be something in mundane that did the "right" thing for this case, is it gone?

**Comment by markhibberd** [comment]
:+1: regardless of what we do with mundane.

**Pull request merged**